src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,28 @@
+package step2
+
+class Operator(type: String) {
```

Comment:
`Operator`는 `Type`과 합칠 수 있을까요?

```suggestion
enum class Operator(private val symbol: String, val operation: (Double, Double) -> Double) {
```

Reply:
enum class로 변경적용하였습니다. 항상 더 간결하게 쓸 수 있는 방법이 있는 것 같네요